src/components/CreateFamily/FamilyForm/CreateFamilyModal.cy.tsx

```diff
@@ -0,0 +1,63 @@
+import { noop } from '@fullstacksjs/toolbox';
+
+import { CreateFamilyModal } from './CreateFamilyModal';
+
+const requiredFieldMessage = 'این فیلد ضروری است';
+const minLengthMessage = 'نام خانواده باید حداقل ۵ حرف باشد';
+const buttonSelector = '[data-test="submit-button"]';
+const inputSelector = '[data-test="family-name"]';
+
+describe('CreateFamilyModal', () => {
```

Comment:
I recommend putting space between words and your tests are about `form` not `modal`